### DISCUSSION TITLE
How to draw tree-table?

### DISCUSSION THREAD

**Discussion by Lapshin** [Textualize/textual#3740] [Q&A] [answered]
How can I implement something like this https://phillbeck.github.io/react-gridtree-example using textual?

![image](https://camo.githubusercontent.com/b8a80b161d1bba9439390924a30d099336622e46623dfd4cb09991720839a313/68747470733a2f2f692e696d6775722e636f6d2f503251657348682e706e67)

**Comment by davep** [comment]
At first glance, it looks similar to something you could build around [`Collapsible`](https://textual.textualize.io/widgets/collapsible/). Failing that, I imagine something similar to this could be built with the [line API](https://textual.textualize.io/guide/widgets/#line-api).

**Comment by Lapshin** [comment, marked as answer]
In case somebody is looking for a solution - check https://github.com/Lapshin/mapuche
